qiskit_experiments/library/characterization/t2ramsey.py

```diff
@@ -91,10 +87,13 @@ def __init__(
         Args:
             qubit: the qubit under test.
             delays: delay times of the experiments.
-            unit: Optional, time unit of `delays`.
+            unit: Optional, time unit of `delays`. \
```

Comment:
This doesn't block but in the Google docstring guide we use indented linefeed rather than backslash.
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Fixed, as you suggested.